packages/create-cloudflare/scripts/common.ts

```diff
@@ -18,6 +18,8 @@ export type Worker = {
 	created_on: string;
 };
 
+const tmpPrefixes = ["tmp-stratus-e2e-", "tmp-wrangler-e2e-", "tmp-c3-e2e-"];
```

Comment:
Alternatively, one could name these: "tmp-e2e-stratus-...", "tmp-e2e-wrangler-...", etc and then we only need to check for "tmp-e2e-" as a prefix.